docs/core/testing/unit-testing-platform-faq.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Microsoft.Testing.Platform FAQ
+description: Get answers to frequently asked questions about Microsoft.Testing.Platform.
+author: Evangelink
+ms.author: amauryleve
+ms.date: 07/09/2024
+---
+
+# Microsoft.Testing.Platform FAQ
+
+This article contains answers to commonly asked questions about `Microsoft.Testing.Platform`.
+
+## error CS8892: Method 'TestingPlatformEntryPoint.Main(string[])' will not be used as an entry point because a synchronous entry point 'Program.Main(string[])' was found
+
+When you are manually defining an entry point (`Main`), there will be a conflict with the entry point generated by `Microsoft.Testing.Platform`. To avoid this issue you can either:
```

Comment:
```suggestion
Manually defining an entry point (`Main`) in test project, or referencing a test project into an application that already has an entry point, will result in conflict with the entry point generated by `Microsoft.Testing.Platform`. To avoid this issue take one of these steps:
```